.github/workflows/claude-nl-suite.yml

```
@@ -0,0 +1,543 @@
+name: Claude NL/T Full Suite (Unity live)
+
+on:
+    workflow_dispatch: {}
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)